frontends/web/src/api/exchanges.ts

```diff
@@ -63,18 +63,31 @@ export const getMoonpayBuyInfo = (code: string) => {
   };
 };
 
+export type ExchangeErrorCode = 'userAbort' | 'addressNotFound';
```

Comment:
`''` empty code missing.

Reply:
I made `errorCode` optional in `AddressSignResponse`

Reply:
That doesn't reflect the backend accurately. `?: ...` means it can be undefined, but the backend never returns undefined (for defined struct fields). It returns the empty string `''` in this case. For pointers, `null` is returned (which is different from `undefined` :sweat_smile:).

Reply:
I am not sure about this. I think that the `omitempty` tag (see https://github.com/Beerosagos/bitbox-wallet-app/blob/address-verification/backend/handlers/handlers.go#L1168-L1169) causes the backend to avoid the error fields in the returned json. In my tests, in case of success I just get `{"success":true}`. Am I missing something?